iree/compiler/setup.py

```diff
@@ -0,0 +1,357 @@
+# Copyright 2021 The IREE Authors
```

Comment:
2022

Reply:
Git didn't pick it up as a rename b/c I made substantial changes, but it is and should be original date.